Learning_test_automation/Spotkanie8_Zadania/SeleniumCsharp/SeleniumCsharp/SeleniumCsharp.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Moznaby rozbic projekt `SeleniumCsharp`, bo teraz jest to projekt testowy pod NUnita + page objecty + selenium

Reply:
W sensie rozbić na dwa projekty?

Reply:
tak